lib/manageiq/active_record_connector.rb

```diff
@@ -41,7 +41,8 @@ def self.establish_connection_if_needed(db_config, log_path = nil)
 
       unless existing_connection
         ActiveRecord::Base.logger ||= get_logger_from log_path
-        ActiveRecord::Base.establish_connection(db_config)
+        ActiveRecord::Base.configurations = connection_configurations_from db_config
+        ActiveRecord::Base.establish_connection
```

Comment:
@jrafanie `configurations` (with an "s"), but yes, that is basically the gist of it.  I could have made it longer by complaining about the fact that `ActiveRecord` is too tightly coupled to `Rails` in this regard... but I digress...